trogdor/README.md

```diff
@@ -6,13 +6,8 @@ Trogdor can run benchmarks and other workloads.  Trogdor can also inject faults
 
 Quickstart
 =========================================================
-First, we want to start a single-node Kafka cluster with a ZooKeeper and a broker.
+First, we want to start a single-node Kafka cluster with one broker.
```

Comment:
We need to update this documentation to reflect KRaft mode. Perhaps you can add a reference to [this section](https://github.com/apache/kafka/blob/trunk/README.md?plain=1#L93) of the README.